content/en/monitors/notify/notification_rules.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Notification Rules
+further_reading:
+- link: "/monitors/notify/"
+  tag: "Documentation"
+  text: "Configure monitor notifications"
+- link: "/monitors/settings/"
+  tag: "Documentation"
+  text: "Monitor Settings"
+---
+
+## Overview
+
+Monitor notification rules are predefined sets of conditions that automate the process of alerting your team based on predefined conditions and tags. Instead of configuring notification recipients and routing for every monitor individually, Notification rules allow you to define the notification logic and recipients in one place and automatically route all monitor events with matching tags to that list of handles.
+
+## Creating notification rules
+
+<div class="alert alert-warning">You must  have  <a href="/account_management/rbac/permissions/#monitors"><code>monitor_config_policy_write</code> permission</a> to create a rule.</div>
+
+1. Navigate to [**Monitors > Settings > Notification Rules**][1].
+1. Click **New Rule**.
+1. Set the scope for the rule through specific tags. Notification rules use an AND logic for multiple tags. For an example of this, see [Routing logic](#routing-logic).
+1. Add up to 50 notification recipients. Notifications can be sent to emails, Team channels, or Integration channels. For more information, see [Notifications][2].
+1. Add a name for the rule.
+1. Click **Create Rule**.
+
+{{< img src="/monitors/notifications/notification_rules/notification_rules_config.png" alt="Configuration for a notification rule, showing tag scopes, recipients, and matching monitors" style="width:100%;" >}}
+
+## Managing notification rules
+
+### From Monitor Settings
+
+{{< img src="/monitors/notifications/notification_rules/settings_notification_rules.png" alt="List of notification rules in Monitor Settings" style="width:100%;" >}}
+
+The [Monitor Notification Rules][1] page displays a table of all your notification rules with the following columns:
+
+- **Scopes**: Shows the tag combinations that define when this rule applies (for example, `team:shopist service:web-store env:prod`).
+- **Teams**: Lists the teams that this notification rule is associated with (available only when the team tag is added in the scope)
+- **Coverage**: Shows the number of monitors that match this rule's scopes. Use this to verify rule coverage and identify rules that need adjustment.
+- **Recipients**: Lists the notification channels (such as Slack or email) that will receive alerts when this rule matches.
+- **Actions**: Provides options to edit or delete each notification rule. Click the vertical three-dot menu on the notification rule and select **Edit** or **Delete**.
```

Comment:
I can see why Actions is here, but since it's technically not a column, it feels a bit misleading to list it here as such. I would suggest adding a header for **Managing notification rules** or something like that.